Dockerfile

```diff
@@ -44,15 +30,24 @@ RUN apk update && \
     && echo "gem: --no-ri --no-rdoc" > /etc/gemrc \
     && gem update --system \
     && gem install bundler \
-    && bundle install --system $BUNDLER_ARGS \
-    && apk del .ruby-builddeps \
-    && rm -rf /var/cache/apk/*
+    && bundle install --clean --no-cache --system $BUNDLER_ARGS
```

Comment:
Had a look at the generated image and it looks like there is currently an issue with bundler

Can you please add the following line to this command?

```
  # temp fix for https://github.com/bundler/bundler/issues/6680
  rm -rf /usr/local/bundle/cache
```